docs/UserGuide.md

```diff
@@ -358,6 +358,14 @@ Examples:
 
 </box>
 
+<box type="warning">
+
+##### Warning
+
+Filtering the order list by status and changing an order's status will cause the order to not be displayed under the filtered list.
+
```

Comment:
Maybe can give example? Like for example filter to completed and then unmark it or something like that

Reply:
Then can move all the warning statements into a separate warning box? Other commands just have warnings as part of the "constraints" box

Reply:
this is not like giving warning in the GUI. This is warning to the user

Reply:
> Maybe can give example? Like for example filter to completed and then unmark it or something like that

Will add example.



> Then can move all the warning statements into a separate warning box? Other commands just have warnings as part of the "constraints" box

Like foochao said, this is a warning in the user guide rather than GUI. It's mainly to let the reader know that this is a thing.